wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -74,17 +75,19 @@ static int _wpa_ctrl_command(struct wpa_ctrl *ctrl, const char *cmd, int print,
 		resp[len - 1] = '\0';
 	}
 
-	if (print) {
+	if (!print_ok) {
```

Comment:
This would print `OK` only once, but what we want is:

* `wpa_cli` should still output OK/FAIL for each command (standard behaviour as this is interactive)
* Everything else i.e., `wifi` commands should never output OK/FAIL, not even once.